finance/go-ethereum/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem 1.0
+
+name				go-ethereum
+version				1.6.6
+revision			0
+categories			finance crypto
+license				GPL-3
+platforms			darwin
+maintainers			openmaintainer krischik
```

Comment:
The maintainer option should now be written like this to include both emails and GitHub usernames:
```
maintainers {krischik @krischik} openmaintainer
```